common/src/main/java/org/apache/sedona/common/Functions.java

```diff
@@ -1227,6 +1228,10 @@ public static Geometry minimumBoundingCircle(Geometry geometry, int quadrantSegm
     return circle;
   }
 
+  public static Geometry orientedEnvelope(Geometry geometry) {
```

Comment:
do you mind also porting this to Flink and Snowflake?

Reply:
Added Flink and Snowflake implementations